code/modules/power/lighting.dm

```diff
@@ -9,6 +9,7 @@
 #define LIGHT_BROKEN 2
 #define LIGHT_BURNED 3
 
+GLOBAL_VAR_INIT(emergency_light, 0)
```

Comment:
Instead of a new global var make an actual alert for this or use an existing one for these emergency lights